pipeline/liminal.yml

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+name: GettingStartedPipeline
+volumes:
+  - volume: gettingstartedvol
+    claim_name: gettingstartedvol-pvc
+    local:
+      path: .
+images:
+  - image: mmrs-python
+    type: python
+    source: .
```

Comment:
The build of images is triggered by `liminal build`.  It used the directive images in liminal.yml like above.  IIt seems that as I specified python type, liminal automatically install libs from requirements.txt and as source is the current directory ,it is added inside the docker image. 
However i don't deep dive because it is my first time to use the tool and choose it because it's easy to setup/run.